### DISCUSSION TITLE
Part of Chinese document misleads me...

### DISCUSSION THREAD

**Discussion by Zenyet** [Milkdown/milkdown#681] [Ideas]
<img width="617" alt="截屏2022-08-17 下午3 20 39" src="https://user-images.githubusercontent.com/52519506/185058851-5267cd03-1117-4a46-9e82-2f2f42921b28.png">

As you see, i try it, but actually it must import @milkdown/preset-commonmark!!  😭

**Comment by Saul-Mirone** [comment]
Most of users will install them, but there're still some users may write themes and presets on their own.
